lexicons/app/bsky/moderation/defs.json

```diff
@@ -0,0 +1,95 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.moderation.defs",
+  "defs": {
+    "modServiceView": {
+      "type": "object",
+      "required": ["uri", "cid", "creator", "indexedAt"],
+      "properties": {
+        "uri": { "type": "string", "format": "at-uri" },
+        "cid": { "type": "string", "format": "cid" },
+        "creator": { "type": "ref", "ref": "app.bsky.actor.defs#profileView" },
+        "description": {
+          "type": "string",
+          "maxGraphemes": 300,
+          "maxLength": 3000
+        },
+        "descriptionFacets": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "app.bsky.richtext.facet" }
+        },
+        "likeCount": { "type": "integer", "minimum": 0 },
+        "viewer": { "type": "ref", "ref": "#modServiceViewerState" },
+        "indexedAt": { "type": "string", "format": "datetime" },
+        "labels": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "com.atproto.label.defs#label" }
+        }
+      }
+    },
+    "modServiceViewDetailed": {
+      "type": "object",
+      "required": ["uri", "cid", "creator", "policies", "indexedAt"],
+      "properties": {
+        "uri": { "type": "string", "format": "at-uri" },
+        "cid": { "type": "string", "format": "cid" },
+        "creator": { "type": "ref", "ref": "app.bsky.actor.defs#profileView" },
+        "description": {
+          "type": "string",
+          "maxGraphemes": 300,
+          "maxLength": 3000
+        },
+        "descriptionFacets": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "app.bsky.richtext.facet" }
+        },
+        "policies": {
+          "type": "ref",
+          "ref": "app.bsky.moderation.defs#modServicePolicies"
+        },
+        "likeCount": { "type": "integer", "minimum": 0 },
+        "viewer": { "type": "ref", "ref": "#modServiceViewerState" },
+        "indexedAt": { "type": "string", "format": "datetime" },
+        "labels": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "com.atproto.label.defs#label" }
+        }
+      }
+    },
+    "modServiceViewerState": {
+      "type": "object",
+      "properties": {
+        "like": { "type": "string", "format": "at-uri" }
+      }
+    },
+    "modServicePolicies": {
+      "type": "object",
+      "required": ["reportReasons", "labelValues"],
+      "properties": {
+        "description": {
+          "type": "string",
+          "maxGraphemes": 10000,
+          "maxLength": 100000
+        },
+        "descriptionFacets": {
+          "type": "array",
+          "items": { "type": "ref", "ref": "app.bsky.richtext.facet" }
+        },
```

Comment:
Gonna suggest we drop the `policies.description` here until we feel more confident about how we're representing documents